avocado/plugins/runners/exec_test.py

```diff
@@ -119,7 +123,11 @@ def _is_uri_a_file_on_cwd(uri):
         return False
 
     def _get_env(self, runnable):
-        env = dict(os.environ)
+        clear_env = runnable.config.get("runner.exectest.clear_env", False)
+        if clear_env:
+            env = {}
+        else:
+            env = dict(os.environ)
```

Comment:
I don't know enough about the surrounding code to know if this clear is done before or after inserting the Avocado standard environment variables.  i.e. will the final environment have (1) only have variables explicitly set via kwargs, or (2) things explicitly set in kwargs + things that Avocado sets.  I can actually see uses for either case, as well as the original case (3) things set explicitly in kwargs + things set by Avocado + whatever was in the environment to begin with.

Reply:
Right now, it does the number 2. It will clear environment before adding kwargs and Avocado defaults.

Reply:
Sure, we can add (1) as well. I will come up with solution.